Kitodo-XML-SchemaConverter/src/test/java/org/kitodo/xmlschemaconverter/XmlSchemaConverterTest.java

```diff
@@ -90,6 +88,58 @@ public void shouldConvertToInternalFormat() throws IOException, ParserConfigurat
         Assert.assertEquals("PublicationYear after conversion is wrong!", year, "1999-12-31");
     }
 
+    @Test
+    public void shouldConvertMarcToInternalFormat() throws IOException, ParserConfigurationException, SAXException {
+        DataRecord testRecord = new DataRecord();
+        testRecord.setMetadataFormat(MetadataFormat.MARC);
+        testRecord.setFileFormat(FileFormat.XML);
+
+        DataRecord internalFormatRecord;
+
+        try (InputStream inputStream = Files.newInputStream(Paths.get(MARC_TEST_FILE_PATH))) {
+            testRecord.setOriginalData(IOUtils.toString(inputStream, Charset.defaultCharset()));
+            internalFormatRecord = converter.convert(testRecord, MetadataFormat.KITODO, FileFormat.XML, null);
+        }
+
+        Assert.assertNotNull("Conversion result is empty!", internalFormatRecord);
+        Assert.assertEquals("Conversion result has wrong MetadataFormat!",
+                internalFormatRecord.getMetadataFormat(), MetadataFormat.KITODO);
```

Comment:
You confused "expected" and "actual". 
Usually it's first "expected" then "actual". Please change, to avoid missleading Error messages"

As I see it, it's in the whole test-file